DMX.Gatekeeper.Api/Models/Commands/CommandType.cs

```diff
@@ -0,0 +1,12 @@
+﻿// ---------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// ---------------------------------------------------------------
+
+namespace DMX.Gatekeeper.Api.Models.Commands
+{
+    public enum CommandType
+    {
+        Download,
+        Execute

```

Comment:
Let's put `Execute` above `Download` because `Execute` is more likely to be the default 0 more than any other command. It's inclusive to all commands.